qa/L0_s3_local/test.sh

```diff
@@ -71,15 +71,19 @@ export AWS_ACCESS_KEY_ID=minio && \
 
 # create and add data to bucket
 python -m pip install awscli-local && \
-    awslocal --endpoint-url=http://localhost:4572 s3 mb s3://demo-bucket && \
+    awslocal --endpoint-url=http://localhost:4572 s3 rm s3://demo-bucket && \
+    awslocal s3 rb s3://demo-bucket && \
+    awslocal s3 mb s3://demo-bucket && \
     awslocal s3 sync $DATADIR s3://demo-bucket
 
 RET=0
 
 # Test with hostname
-SERVER_ARGS="--log-verbose=1 --model-repository=s3://localhost:4572/demo-bucket"
+SERVER_ARGS="--model-repository=s3://localhost:4572/demo-bucket --allow-poll-model-repository=false --allow-model-control=true"
```

Comment:
--model-control-mode=none is the new way to turn off polling. The flags you are using will be removed in Triton V2.